src/Event/EnhancedEcommerceEvents.php

```diff
@@ -30,8 +31,9 @@
   const ALTER_EVENT_DATA = 'commerce_google_tag_manager.alter_event_data';
 
   /**
-   * Event fired when tracking a checkout step. This allows event listeners
-   * to track additional checkout step options.
+   * Event fired when tracking a checkout step.
+   *
+   * This allows event listeners to track additional checkout step options.
```

Comment:
Do you agree with this change ? 

The first comment must be a one line comment, the second can be multiline (drupal 8 coding standard).